packages/react-liveness/src/components/FaceLivenessDetector/shared/FaceLivenessErrorModal.tsx

```diff
@@ -69,9 +69,14 @@ const renderToastErrorModal = (props: {
     <>
       <Flex className={LivenessClassNames.ErrorModal}>
         <AlertIcon ariaHidden variation="error" />
-        <Text className={LivenessClassNames.ErrorModalHeading}>{heading}</Text>
+        <Text
+          className={LivenessClassNames.ErrorModalHeading}
+          id="amplify-liveness-error-heading"
+        >
+          {heading}
+        </Text>
       </Flex>
-      {message}
+      <div id="amplify-liveness-error-description">{message}</div>
```

Comment:
Correct me if I am wrong, but it looks like `message` will always be a `string` so maybe we should use a `Text` element here? Also, is there a reason to use "description" instead of "message" in the `id`?

Reply:
Yeah that would make sense. And not really, I was thinking it was a description of the error but I can change to message.